ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -240,7 +240,6 @@ public static ScheduledExecutorService newScheduler(int numThreads, String threa
         new ScheduledThreadPoolExecutor(numThreads, new SchedulerThreadFactory(threadNamePrefix, isDaemon));
     scheduler.setContinueExistingPeriodicTasksAfterShutdownPolicy(false);
     scheduler.setExecuteExistingDelayedTasksAfterShutdownPolicy(false);
-    scheduler.setRemoveOnCancelPolicy(true);
```

Comment:
Removed since we are not constantly cancelling and adding new runnable to the scheduler.